src/works/index.ts

```diff
@@ -69,6 +78,15 @@ export const getWork = async ({
     },
     publishedAt: getWorkPublishDate(workPage),
     updatedAt: getWorkUpdateDate(workPage),
+    chapter:
```

Comment:
I think we probably want to call this `chapterInfo` to be more descriptive for what the data it actually holds. This will be super useful for end users who are using typescript and are getting hints from their editor.